code/dijkstra-algorithm/Infrastructure/src/main/java/ru/unn/agile/dijkstraalgorithm/infrastructure/TextLogger.java

```diff
@@ -0,0 +1,68 @@
+package ru.unn.agile.dijkstraalgorithm.infrastructure;
+
+import ru.unn.agile.dijkstraalgorithm.viewmodel.ILogger;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.List;
+import java.util.Locale;
+
+public class TextLogger implements ILogger {
+    private final String filename;
+    private final BufferedWriter writer;
+    private static final String DATE_FORMAT_POINT = "yyyy-MM-dd HH:mm:ss";
+
+    private static String now() {
+        Calendar cal = Calendar.getInstance();
+        SimpleDateFormat sdf = new SimpleDateFormat(DATE_FORMAT_POINT, Locale.ENGLISH);
+        return sdf.format(cal.getTime());
+    }
+
+    public TextLogger(final String filename) {
+        this.filename = filename;
+
+        BufferedWriter bufferedLogWriter = null;
+        try {
+            bufferedLogWriter = new BufferedWriter(new FileWriter(filename));
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+        writer = bufferedLogWriter;
+    }
+
+    @Override
+    public void log(final String s) {
+        try {
+            writer.write(now() + " > " + s);
```

Comment:
Move `>` separator to private static final field

Reply:
What for?

Reply:
Done